src/Network/Network/PrivateLinkService/PrivateLinkServiceProvider/ProviderConfiguration.cs

```diff
@@ -17,7 +17,7 @@ static ProviderConfiguration()
             RegisterConfiguration("Microsoft.ApiManagement/service", "2021-04-01-preview", true, true);
             RegisterConfiguration("Microsoft.AppConfiguration/configurationStores", "2020-06-01", true, true);
             RegisterConfiguration("Microsoft.Attestation/attestationProviders", "2020-10-01", true, true);
-            RegisterConfiguration("Microsoft.Authorization/resourceManagementPrivateLinks", "2020-05-01", false);
+            RegisterConfiguration("Microsoft.Authorization/resourceManagementPrivateLinks", "2020-05-01", true, true);
```

Comment:
Hi @Vishakha-Git , could you pull latest main code into your branch? Current we have three flags to be filled when register configuration.